desktop/assets/css/desktop.css

```diff
@@ -317,7 +317,7 @@ div.ui-resizable-handle {
   */
   margin: 0 0 0 0px;
   padding: 0 0 0 0px;
-  min-height: 77%;
+  /* min-height: 77%; NO!! */
```

Comment:
@CDWimmer - Everything is looks GOOD except without this line the chat window doesnt fill unless it hits enough messages.

Makes the default state of fresh chat window look odd ( screenshot attached )


<img width="593" alt="Screen Shot 2022-04-11 at 6 18 23 PM" src="https://user-images.githubusercontent.com/70011/162842666-5abdddd2-217b-4a7f-8654-90ffadaa2b2d.png">

Reply:
Dang it, thought I'd fixed that, must've not transferred to buddy chat.